sql/driver/conn.go

```diff
@@ -112,6 +109,9 @@ func (c *conn) send(args Request) (*rows, error) {
 		return r, nil
 	}
 	result := resp.Results[index]
+	if result.Error != nil {
```

Comment:
Ditto. I would think we should loop over the results and return the first error that occurred.